docs/snippets/common/badge-story-custom-argtypes.mdx.mdx

```diff
@@ -0,0 +1,58 @@
+```md
+<!-- Badge.stories.mdx -->
+
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
+
+import { Badge } from './Badge';
+
+<Meta
+  title="MDX/Badge"
+  argTypes={{
+    status: {
+      name: 'Badge Status',
+      description: 'Available options available to the Badge',
+      control: {
+        type: 'select',
+        options: [
+          'positive',
+          'negative',
+          'warning',
+          'error',
+          'error',
+          'neutral'
+        ],
+      },
+      table: {
+        defaultValue: {
+          summary: 'positive'
+        },
+        type: {
+          summary: 'Shows options to the Badge',
+          detail: 'Listing of available options'
+        },
+      },
+    },
+    label: {
+      name: 'Badge Content',
+      description: 'Text shown by Badge',
+      control: {
+        type: 'text'
+      },
+      table: {
+        type: {
+          summary: 'The label contents',
+          detail: 'Text displayed by the Badge'
+        }
+      }
+    }
+  }}
+/>
+
+### Badge
+
+Let's define a story for our `Badge` component
+
+<ArgsTable of={Badge} />
```

Comment:
This won't show controls because there's no story to control. Is that intentional?

Reply:
@shilman the way it's presented it's intentional. I went with a focus on how the reader can use argTypes in their mdx stories only